src/marshaller/HipieDDL.js

```diff
@@ -114,6 +114,14 @@
     }
     ChoroMappings.prototype = Object.create(SourceMappings.prototype);
 
+    function HeatMapMappings(visualization, mappings) {
+        SourceMappings.call(this, visualization, mappings);
+        this.columns = ["x", "y", "weight"];
+        this.columnsIdx = { x: 0, y:1, weight: 2 };
+        this.init();
+    }
+    HeatMapMappings.prototype = Object.create(SourceMappings.prototype);
```

Comment:
This is ok for now - but we will need to define a new "data shape" for this (which will also be shared with Bubble Charts).  At which point this method will be renamed to match that data shape name - "XYWChart" or such like.